lib/client/checkMimeType.js

```diff
@@ -0,0 +1,15 @@
+const Client = require('../client/constructor');
+/**
+ * Checks if a Base64 string contains any of the predefined MIME types.
+ *
+ * @param {string} base64String - The Base64 encoded string with a MIME type prefix.
+ * @returns {boolean} - Returns true if any predefined MIME type is found, false otherwise.
+ */
+Client.prototype.check_mime_type = function (base64String) {
+    const mimeTypes = [
+        'image/png',
```

Comment:
I think there's more, like webp. You can just check if it starts with `data:` and has `;base64,`